patch_common/include/patch_common/CodeInjection.h

```diff
@@ -15,6 +15,7 @@ class BaseCodeInjection : public Installable
     uintptr_t m_addr;
     subhook::Hook m_subhook;
     CodeBuffer m_code_buf;
+    bool m_needs_trampoline;
```

Comment:
I'm not opposed to it, but can you help me understand what this is improving? Is there an instance currently where we would expect `m_needs_trampoline` to ever be false?